extensions/amp-sidebar/0.1/amp-sidebar.md

```diff
@@ -326,6 +326,49 @@ When using `toolbar` feature, `autoscroll` only works if the `<nav toolbar>` ele
 
 Please see [this example file](https://github.com/ampproject/amphtml/blob/master/examples/amp-sidebar-autoscroll.amp.html) for a working example code.
 
+## Building Nested Menus
+
+`<amp-sidebar>` can supports drilldown (nested) menus through a child component named [`<amp-nested-menu>`](https://amp.dev/documentation/components/amp-nested-menu/). With `<amp-nested-menu>`, `<amp-sidebar>` can support nesting one or more layers of submenus (and transition between them) as demonstrated by the following example:
+
+[tip type="note"]
+We recommend wrapping every menu item in a `li` element to improve accessibility and keyboard support.
```

Comment:
Can this be added to the documentation under `Accessibility` in the `amp-nested-menu` ref doc? 

Also, suggestion to drop "we recommend" and just say "Wrap every menu item in...."

Reply:
That note is the only a11y related paragraph so I didn't make a section for it, maybe later when we have more a11y related documentation

Reply:
While the information is minimal, it is still important. We're trying to guide AMP developers into making their sites more acess8ible, and having ally related information under a header is going to make this much more clear. I would argue that we still include it under the `Accessibility` header in `amp-nested-menu`.

Reply:
Done!